### ISSUE TITLE
Spotify Lyrics helper remained after uninstallation

### ISSUE THREAD

**Issue by MarcAnt01** [JakubSteplowski/SpotifyLyricsNET#15]
I uninstalled the app, but the helper is still installed and there's no way to remove it

**Comment by JakubSteplowski** [comment]
Hi,

There is no such thing as "no way".

I'm sorry for the inconvenience and I plan to add an uninstaller in the next update, in the meantime these are the steps to remove the helper:
1.  Open your task manager.
2.  Go to "Details" and end the process "Spotify_Lyrics_NET_Helper_UWP.exe" (or ""Spotify_Lyrics_NET_Helper.exe" for Win32 version).
3.  Go to "Startup" and disable "Spotify Lyrics .NET Helper UWP" (or "Spotify Lyrics .NET Helper" for Win32 version).
4.  Open the explorer and go to your Documents, then delete the directory "Spotify Lyrics .NET" (or just the helper to preserve the lyrics you marked as "correct").

Let me know if you have any problem with the uninstallation process.

**Comment by MarcAnt01** [comment]
It worked properly, but still weird to me that a Microsoft Store app doesn't get uninstalled from start menu and I guess it's the same for most users

**Comment by JakubSteplowski** [comment]
> It worked properly, but still weird to me that a Microsoft Store app doesn't get uninstalled from start menu and I guess it's the same for most users

The reason is that this isn't a native UWP app, it's a Win32 app ported to UWP using Microsoft Desktop Bridge.
The helper is a companion software included in the main program, so it's not reachable by the uninstalling process generated by Microsoft.

I will think about moving to MSIX which is a new way to port Win32 programs to UWP which, in theory, includes better support for this kind of things, or as I said before I will include an uninstaller to make this automatic.

**Comment by MarcAnt01** [comment]
Yeah, I know the difference, but if I install Telegram Desktop or ShareX from the Store and I uninstall them from start there isn't anything left and I expect this behavior with anything installed from there.
